File: rizzbuzz.py
def rizzbuzz(end_num):
    for i in range(1, end_num + 1):
        if i % 3 == 0 and i % 5 == 0:
            print(f"{i} rizzbuzz <3   HELLA RIZZZ!!!")
        else:
            if i % 3 == 0:
                print(f"{i} rizz")

            elif i % 5 == 0:
                print(f"{i} buzz")

            else:
                print(f"{i}")


if __name__ == "__main__":
    rizzbuzz(100)

File: rizzbuzz.js
function rizzbuzz(end_num) {
  for (let i = 1; i <= end_num; i++) {
    if (i % 3 == 0 && i % 5 == 0) {
      console.log(`${i} rizzbuzz <3   HELLA RIZZZ!!!`);
    } else {
      if (i % 3 == 0) {
        console.log(`${i} rizz`);
      } else if (i % 5 == 0) {
        console.log(`${i} buzz`);
      } else {
        console.log(i);
      }
    }
  }
}

rizzbuzz(100);
